package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "chss-blog",
+  "version": "0.1.0",
+  "description": "CHSS Phoenix - Blog",
+  "main": "index.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/CHSSPhoenix/chss-blog.git"
```

Comment:
Hi @paolo, I think you have a typo here it should be https://github.com/CHSSPhoenix/chss-blog.git without git+, correct?

Reply:
I believe it's npm's notation, it was automatically filled for me when I executed `npm init`